komorebi-core/src/lib.rs

```diff
@@ -88,6 +88,8 @@ pub enum SocketMessage {
     EnsureWorkspaces(usize, usize),
     EnsureNamedWorkspaces(usize, Vec<String>),
     NewWorkspace,
+    ToggleTaskbarVisibility,
+    ShowTaskbar(bool),
```

Comment:
Let's change this to `Taskbar(Taskbar)` where the struct is generated by [`gen_enum_subcommand_args!`](https://github.com/LGUG2Z/komorebi/blob/e40fa11b0d18387d32d2ef44ef5b8dcbeb5b8d4e/komorebic/src/main.rs#L152) using `Taskbar: BooleanState`; this way the call signature will match `focus-follows-mouse` (eg. `komorebic focus-follows-mouse disable`, `komorebic taskbar disable`)